Item/Item.js

```diff
@@ -20,11 +20,13 @@ import Slottable from '@enact/ui/Slottable';
 import PropTypes from 'prop-types';
 import compose from 'ramda/src/compose';
 
-import {Marquee, MarqueeController} from '../Marquee';
+import {MarqueeBase, MarqueeController, MarqueeDecorator} from '../Marquee';
 import Skinnable from '../Skinnable';
 
 import componentCss from './Item.module.less';
 
+const Marquee = MarqueeDecorator({className: componentCss.marquee}, MarqueeBase);
```

Comment:
Could you leave the reason why you change this line on the PR description?

Reply:
I added it